PLib/Codex/PCodex.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Klei;
+using Harmony;
+
+namespace PeterHan.PLib.Codex {
```

Comment:
Making a namespace for the Codex alone seems a bit overkill, but how about making it a Datafiles namespace? Then the localization changes can go in that namespace as well.